### PR TITLE
Add Banglalink logo and link 2 app

### PR DESCRIPTION
**Pull request by thekeyboardworrier**
## Description

Added 1 icon and linked MyBL and MyBL Lite with the new icon.

## Type of change

:x: Bug fix (non-breaking change which fixes an issue)
:white_check_mark: Icon addition (non-breaking change that adds/modifies Lawnicons's icons)
:white_check_mark: General change (non-breaking change that doesn't fit the above categories like copyediting)

### Icons added
* MyBL (`com.arena.banglalinkmela.app`)

### Icons linked
* MyBL (linked `com.arena.banglalinkmela.app` to `@drawable/banglalink`)
* MyBL Lite (linked `net.banglalink.mybllite` to `@drawable/banglalink`)

### CODE REVIEW

File: svgs/banglalink.svg
Comment (Chefski) [inline svgs/banglalink.svg:1]: This icon won't be suitable as it has varying widths, some looking less than 6px. Please try redesign it. Maybe a `b` can be used instead?

**Review by Chefski — CHANGES_REQUESTED** [review]

File: svgs/banglalink.svg
Comment (thekeyboardworrier) [reply svgs/banglalink.svg:1]: Changed the logo. It's now a single `b`.

Comment (Chefski) [reply svgs/banglalink.svg:1]: My review seems to not have sent yesterday. 

One of the rectangles has a stroke of 6.04. Could you make it 6px? I'd change the rectangles to be a line path instead though as it'll be simpler. If I remember correctly, the circle goes a bit over the line on the left. Mind moving it slightly?

**Review by Chefski — CHANGES_REQUESTED** [review]

File: svgs/banglalink.svg
Comment (thekeyboardworrier) [reply svgs/banglalink.svg:1]: Changed it again. Thank you.

**Review by Chefski — APPROVED** [review]
Cool. Thanks :)

**Pull request merged**